### PR TITLE
docs: add missing commas

### PR DESCRIPTION
**Pull request by cipriansauliuc**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

The Oxford comma is used in some lists but not in others.

## What is the new behavior?
Use of the Oxford comma in missing lists.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/deprecations.md [hunk 1]
```diff
@@ -640,7 +640,7 @@ The string syntax has been replaced with dynamic import statements.
 The `DeprecatedLoadChildren` type was removed from `@angular/router`.
 Find more information about the replacement in the [`LoadChildrenCallback` documentation](api/router/LoadChildrenCallback).
 
-The supporting classes `NgModuleFactoryLoader`, `SystemJsNgModuleLoader` and `SystemJsNgModuleLoaderConfig` classes were removed from `@angular/core`, as well as `SpyNgModuleFactoryLoader` from `@angular/router`.
+The supporting classes `NgModuleFactoryLoader`, `SystemJsNgModuleLoader`, and `SystemJsNgModuleLoaderConfig` classes were removed from `@angular/core`, as well as `SpyNgModuleFactoryLoader` from `@angular/router`.
```
Comment (bob-watson) [inline aio/content/guide/deprecations.md:643]: `classes` is mentioned before the list, so it's not necessary after the list.
```suggestion
The supporting classes `NgModuleFactoryLoader`, `SystemJsNgModuleLoader`, and `SystemJsNgModuleLoaderConfig` were removed from `@angular/core`, as well as `SpyNgModuleFactoryLoader` from `@angular/router`.
```

File: aio/content/guide/glossary.md [hunk 2]
```diff
@@ -739,7 +739,7 @@ To learn more, see [Introduction to Services and Dependency Injection][AioGuideA
 
 A configuration of [components][AioGuideGlossaryComponent], [directives][AioGuideGlossaryDirective], and [pipes][AioGuideGlossaryPipe] to indicate that this class can be imported directly without declaring it in any [NgModule][AioGuideGlossaryNgmodule].
 
-Standalone components, directives and pipes mainly differ from non-standalone ones by:
+Standalone components, directives, and pipes mainly differ from non-standalone ones by:
```
Comment (bob-watson) [inline aio/content/guide/glossary.md:742]:
```suggestion
Standalone components, directives, and pipes differ from non-standalone ones by:
```

**Review by bob-watson — CHANGES_REQUESTED** [review]
Thanks for the edits!
Just a couple of comments to look into, but other than that, LGTM!

**Review by bob-watson — CHANGES_REQUESTED** [review]
LGTM.
Can you please clean up the branch by squashing the four commits into one and formatting the resulting commit message so that it starts with `docs: `?

Thanks, again, for your contribution!

**Comment by twentyfivedots** [comment]
> 



> LGTM. Can you please clean up the branch by squashing the four commits into one and formatting the resulting commit message so that it starts with `docs: `?
> 
> Thanks, again, for your contribution!

Done. Thanks for the hints!

**Review by bob-watson — APPROVED** [review]
LGTM

Reviewed-for: global-docs-approvers

**Review by AndrewKushnir — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 752e2245f4893573c66203bc733804534707eb2f.

**Pull request closed without merging**